src/main/java/javax/validation/Validation.java

```diff
@@ -150,6 +150,8 @@ public static GenericBootstrap byDefaultProvider() {
 	 * {@link ValidationProvider} implementation of the ACME provider.
 	 *
 	 * @param providerType the {@code ValidationProvider} implementation type
+	 * @param <T> the type of the {@code Configuration} corresponding to this {@code ValidationProvider}
+	 * @param <U> the type of the {@code ValidationProvider} implementation
```

Comment:
Until now I always used link and didn't know about that best practice. Based on this I created a PR that adds the best practice as an addition to the guide: https://github.com/beanvalidation/beanvalidation.org/pull/32